lighthouse-cli/cli-flags.ts

```diff
@@ -104,6 +106,7 @@ Example: --output-path=./lighthouse-results.html`,
 
       // default values
       .default('chrome-flags', '')
+      .default('enable-error-reporting', true)
```

Comment:
> then we won't even prompt? at that point I'd just abandon sentry entirely 😆

arg, I think this is the second time I've mixed this up :)

Taking my confusion as a sign that it's generally confusing (not necessarily true :), and since it's not only encountered reading the code, but also in `--help` as `[default: true]`, would leaving it without a default and prompting on `cliFlags.enableErrorReporting === undefined` in `bin.ts` work instead? Other ideas?